### ISSUE TITLE
Add section on CORS

### ISSUE THREAD

**Issue by willeastcott** [playcanvas/developer.playcanvas.com#218]
It seems that there is an endless stream of developers struggling to handle issues relating to CORS. Despite it not being a PlayCanvas specific issue, we should consider adding a User Manual section on CORS and how to deal with it.

**Comment by yaustar** [comment]
Agreed. Maybe an area about hosting/using files external from where the app is hosted?

Referencing sites like:
https://www.test-cors.org/
https://enable-cors.org/

Will also help.
